clang/lib/Driver/ToolChains/MinGW.cpp

```diff
@@ -15,6 +15,7 @@
 #include "clang/Driver/InputInfo.h"
 #include "clang/Driver/Options.h"
 #include "clang/Driver/SanitizerArgs.h"
+#include "llvm/Config/llvm-config.h" // for LLVM_HOST_TRIPLE
```

Comment:
Possibly we should not have the include for cases like this, where the macro is used directly, not in `#if` and will fail the build when missing?

Reply:
Well, I guess such files (with configuration defines only) make almost no impact in compile time. It seems reasonable to treat _them_ (only?) just as IWYU.

Moreover, this change is a preparation to cleanup. So this include will probably appear here, after removal in other (implicitly used from this file) place.